### ISSUE TITLE
Gpsmon stop working

### ISSUE THREAD

**Issue by aGGreSSiv** [domschl/RaspberryNtpServer#12]
Hello. I had set up two separate servers for both the office and home. I did some operations on one of them, such as setting up a VPN server etc., but I never touched the other one. Suddenly they both switched to Saturn2 last night. When I want to see what's happening with the gpsmon command, I get the following image. It's like they both lost connection with the GPS module at the same time. Or did an update cause this?

![gpsmon](https://github.com/domschl/RaspberryNtpServer/assets/5939750/29ae81f9-ff30-48a2-a0c2-84fda24ccc64)

**Comment by aGGreSSiv** [comment]
Hımm


When I searched a little on the internet and looked for GPSmon problem solution suggestions, I manually typed the port to which your GPS module is connected and followed the suggestion to try it. The problem does not appear. I think GPSmon, which starts automatically, is looking at a different port.

![gpsmon2](https://github.com/domschl/RaspberryNtpServer/assets/5939750/874fee79-ec0e-4e94-a6bb-0d261da5f968)

**Comment by domschl** [comment]
Yes, your first log uses serial port /dev/ttsyS0 and the second working image uses USB /dev/ACM0.

**Comment by aGGreSSiv** [comment]
Yes. That's how it worked until now. Now, when I looked at the ports and decided to try it like this, I took the last screenshot. I don't know much about Linux. Is it something that can change on its own?

**Comment by domschl** [comment]
Unlikely. Maybe because of an update of some config-file? I don't know enough about your environment to make a meaningful suggestion.

**Comment by aGGreSSiv** [comment]
I wish I had asked after doing a little more research. Anyway, maybe if it happens to someone, they will read this and benefit from it. Sorry for taking up your time.
